src/main/java/org/intellij/sdk/codesync/Constants.java

```diff
@@ -21,6 +21,8 @@ private Constants() {
     public static String USER_HOME = System.getProperty("user.home");
     public static String CODESYNC_ROOT =Paths.get(USER_HOME, CODESYNC_DIR_NAME).toString();
     public static String DIFFS_REPO = Paths.get(CODESYNC_ROOT, ".diffs", ".intellij").toString();
+    public static String LOCK_FILE_DIR = Paths.get(CODESYNC_ROOT, ".locks").toString();
+    public static String LOCK_FILE = Paths.get(LOCK_FILE_DIR, ".intellij.yml").toString();
```

Comment:
maybe name this `.daemon.yml` to keep it generic? As I think other IDEs will be using the same file to see if lock is acquired by any other Instance. 
e.g. If PyCharm is running populateBuffer, VSCode should not run it and vice versa.

Reply:
I have renamed the file to `locks.yml` and also removed the `.locks` sub-directory.

Reply:
That sounds good since we'll now have a single file for all locks